### ISSUE TITLE
Code examples generated on html has extra zero width space

### ISSUE THREAD

**Issue by paradite** [PrismJS/prism#1517]
Hi, I raised an issue with redux documentation being unfriendly to copy-pasting: https://github.com/reduxjs/redux/issues/3088

The redux team thinks this is an underlying issue with the syntax highlighting done by PrismJS, that is causing the html rendered to have a `zero width space`, which is invisible to human but readable by parsers such as eslint and prettier (and hence breaking them).

More details and the steps to reproduce can be found here: https://github.com/reduxjs/redux/issues/3088

**Comment by RunDevelopment** [comment]
I will close this issue now because neither can I reproduce the issue now nor was I able to reproduce it in August. It's really hard to follow this up because the Redux team switched to Highlight.js since.
Also, it seems to be highly unlikely that Prism _adds_ any characters on its own.

If the issue should ever come up again, feel free to reopen this.
